eos/rare-b-decays/observables.cc

```diff
@@ -531,6 +531,14 @@ namespace eos
                         Unit::None(),
                         &NonlocalFormFactorObservable<nff::BToKstar, nff::PToV>::abs_H_perp,
                         std::make_tuple("q2")),
+                make_observable("B->K^*::re_Hhat_perp(q2)",  R"(\mathrm{Re}\hat{\mathcal{H}}_\perp^{B \to K^*}(q^2))",
+                        Unit::None(),
```

Comment:
Double check that the Hhat functions are unit less. I doubt it.

Reply:
They are: Blaschke factors, outer functions and H are dimensionless